run-tests.php

```diff
@@ -2885,6 +2886,23 @@ function generate_array_diff(array $ar1, array $ar2, bool $is_reg, array $w): ar
     $old1 = array();
     $old2 = array();
 
+    $format_diff_line = function (int $line_number, string $contents, string $diff_character) use ($colorize): string {
```

Comment:
Can't you move the ``global $colorize;`` inside this function body? As it's currently not used anywhere else by ``generate_array_diff()``

Reply:
I can, I didn't know how much reviewers would care about efficiency. I'd expect repeatedly looking up the same global on every closure call to make this a bit slower for diffs with a large amount of unexpected actual output, but the closure invocation itself would be a bit slower and this probably doesn't need to be optimized

The closure invocation itself would be much slower than the addition of adding a `global $colorize;`, though

Reply:
tbh I didn't think about performance that much, but I could see it having an impact in that case where having it important once be better.